tests/unittests/Foundation/NSHttpCookieTests.mm

```diff
@@ -17,18 +17,19 @@
 #include <TestFramework.h>
 #import <Foundation/Foundation.h>
 
-// Implemented in NSHTTPCookie.mm
-extern void parseCookies(const char* lineptr, id dict);
+// // Implemented in NSHTTPCookie.mm
```

Comment:
was an accident, probably did this temporarily to look at other issues, then forgot about it. ICU has been keeping me distracted for a little while now.

---
In reply to: [72324764](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72324764) [](ancestors = 72324764)